components/text/oniguruma/Makefile

```diff
@@ -0,0 +1,37 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 <contributor name>
+#
+
+BUILD_BITS=32_and_64 # for binaries or 32_and_64 for libraries
+include ../../../make-rules/shared-macros.mk
```

Comment:
The only consumer is jq which is 64-bit only. Do we really want to add a new 32-bit variant? IMO we should try to phase out 32 bit where possible.

Reply:
I have no real opinion on what I want here :shrug:. So I think we can go with yours. I'll send an update later today and make it 64bit only.